backends/graphics/openglsdl/openglsdl-graphics.cpp

```diff
@@ -299,6 +299,9 @@ void OpenGLSdlGraphicsManager::initSize(uint w, uint h, const Graphics::PixelFor
 
 	return OpenGLGraphicsManager::initSize(w, h, format);
 }
+void OpenGLSdlGraphicsManager::setLockedScreen(bool val) {
```

Comment:
Either you put stuff in WindowedGraphicsManager, either you put it in SdlGraphicsManager.
In both case this is useless.